vulture_os/applications/logfwd/config/om_elasticsearch_pre.tpl

```diff
@@ -0,0 +1,88 @@
+{%- if retry_on_els_failures %}
+ruleset(name="{{output_name}}_retry_failure") {
+    action(type="omfile" template="RSYSLOG_DebugFormat" file="/var/log/internal/{{output_name}}_error.log")
+}
+{%- endif -%} {# if retry_on_els_failures #}
+
+ruleset(name="{{output_name}}_retry") {
+    if strlen($.omes!status) > 0 then {
+        # retry case
+        if ($.omes!status == 200) or ($.omes!status == 201) or (($.omes!status == 409) and ($.omes!writeoperation == "create")) then {
+            stop # successful
+        }
+        if ($.omes!writeoperation == "unknown") or (strlen($.omes!error!type) == 0) or (strlen($.omes!error!reason) == 0) then {
+            call {{output_name}}_retry_failure
+            stop
+        }
+        if ($.omes!status == 400) or ($.omes!status < 200) then {
+            call {{output_name}}_retry_failure
+            stop
+        }
+        # else fall through to retry operation
+    }
+    if strlen($.omes!_id) > 0 then {
+        set $.generated_uuid = $.omes!_id;
+    } else {
+        set $.generated_uuid = $uuid;
+    }
+
+    action(type="omelasticsearch"
+            name="{{output_name}}"
```

Comment:
```suggestion
            name="{{output_name}}_retry"
```

Reply:
I had the chance avoiding an Rsyslog error with multiple queues with the same name, good catch !